src/js/pouch.js

```diff
@@ -25,42 +46,79 @@ gpii.pouch.init = function (that) {
     fs.writeFileSync(that.options.pouchConfigPath, JSON.stringify(that.options.pouchConfig, null, 2));
 
     var MemPouchDB = PouchDB.defaults({ db: memdown });
-
-    fluid.each(that.options.databases, function (dbConfig, key) {
-        var db = new MemPouchDB(key);
-        if (dbConfig.data) {
-            var data = require(dbConfig.data);
-            db.bulkDocs(data);
-        }
-    });
-
     that.expressPouchdb = expressPouchdb(MemPouchDB, { configPath: pouchConfigPath });
 
-    //that.expressPouchdb.couchLogger.setFile(logPath);
+    if (PouchDB.isBeingCleaned) {
```

Comment:
This is in theory a good idea, but it seems that you never set this flag.
Rather than failing immediately if pouch is busy (which looks like a virtual certainty given the overall workflow) what I suggest is that you store a promise in isBeingCleaned rather than simply a boolean - and if you discover anything there, you simply delay this function's own promise resolution until it has resolved.